src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -25,7 +25,7 @@
 
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2021, 2023 All Rights Reserved
+ * (c) Copyright IBM Corp. 2021, 2024 All Rights Reserved
```

Comment:
This is a backport of OpenJDK changes, any reason to change the IBM copyright?

Reply:
The change doesn't completely adopt the implementation of that method, we have a different (smaller) change relative to upstream, but I can revert the copyright change if you think that makes more sense.

Reply:
It's not even a backport, it's change to match the upstream OpenJDK code.

Reply:
The IBM part wasn't modified, I don't think the copyright should be updated.